packages/astro/src/content/runtime.ts

```diff
@@ -277,7 +294,9 @@ export function createGetEntry({
 		if (store.hasCollection(collection)) {
 			const entry = store.get<DataEntry>(collection, lookupId);
 			if (!entry) {
-				throw new Error(`Entry ${collection} → ${lookupId} was not found.`);
+				// eslint-disable-next-line no-console
+				console.warn(`Entry ${collection} → ${lookupId} was not found.`);
```

Comment:
There seems to be inconsistency in different parts of the existing code about what the correct behaious should be when `getEntry` is called and there's no match. Should it throw or just warn? I'm inclined to suggest that it throw if there's no matching collection, but just return undefined if there's no matching entry.